### PR TITLE
Doc about how to freeze final grades

### PR DESCRIPTION
**Pull request by giocalitri**
#### What are the relevant tickets?
N/A

#### What's this PR do?
Doc for freezing final grades

#### How should this be manually tested?
Probably try to follow the doc and see if you are successful in freezing some grades for a course run.

### CODE REVIEW

**Comment by pdpinch** [comment]
@annagav can you review this?

File: grades/README.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    python manage.py freeze_final_grades <edx_course_key>
+
+You should receive a confirmation message that an async task has been submitted or an error with the explaination
+of what went wrong.
+
+If you want to check the status of the freezing run
+
+	python manage.py check_final_grade_freeze_status <edx_course_key>
+
+This should tell you how many grades have been frozen and how meny students had an error.
```
Comment (annagav) [inline grades/README.md]: typo meny

File: grades/README.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+To verify this in the heroku bash run
+
+    celery inspect active -A micromasters
+
+You should see each celery worker what is doing. If the tasks are not related to the grades, then celery
+is just processing something else (likely it is refreshing the edX cache or indexing).
+
+If the workers are empty, then something went really wrong and you should probably look at Sentry to see if there is any error there.
+
+Another possibility is that there was an unhandled error for just one (ore very few) student(s).
```
Comment (annagav) [inline grades/README.md]: typo `ore`

File: grades/README.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+### Something went wrong: now what?
+
+There are few places where things might go wrong.
+
+If the task is submitted and nothing happens for a while you might just have the celery workers busy.
+To verify this in the heroku bash run
+
+    celery inspect active -A micromasters
+
+You should see each celery worker what is doing. If the tasks are not related to the grades, then celery
```
Comment (annagav) [inline grades/README.md]: I am not sure this sentence makes sense to me. Could you please rephrase it?

File: grades/README.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+is just processing something else (likely it is refreshing the edX cache or indexing).
+
+If the workers are empty, then something went really wrong and you should probably look at Sentry to see if there is any error there.
+
+Another possibility is that there was an unhandled error for just one (ore very few) student(s).
+
```
Comment (annagav) [inline grades/README.md]: Maybe make those sentences into one paragraph?

**Comment by annagav** [comment]
@giocalitri, also, what are the steps to re-freeze the final grades?

**Comment by giocalitri** [comment]
Good point: I'll add this

File: grades/README.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+Another possibility is that there was an unhandled error for just one (or very few) student(s).
+In that case you will find the error in Sentry, but if you want to force the freezing of the course, just run
+
+	python manage.py complete_course_run_freeze <edx_course_key>
+
+And this will complete the freezing of the course (but it will not solve the problem for the students who got the error).
+
+
+### The grade are frozen, but you changed your mind.
```
Comment (annagav) [inline grades/README.md]: grades?

**Comment by giocalitri** [comment]
done

**Comment by pdpinch** [comment]
@annagav can you take this over and merge it when it's ready?

**Pull request merged**